{"uuid": "917b8aa4-b154-4e63-87f0-5c373672a1c6", "children": ["23aa92d7-88d3-48ee-8231-189af0125063"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF761B8DF85+26693]\n\t(No symbol) [0x00007FF761AEEAD0]\n\t(No symbol) [0x00007FF7619791CA]\n\t(No symbol) [0x00007FF7619CF747]\n\t(No symbol) [0x00007FF7619CF97C]\n\t(No symbol) [0x00007FF761A233C7]\n\t(No symbol) [0x00007FF7619F792F]\n\t(No symbol) [0x00007FF761A200B8]\n\t(No symbol) [0x00007FF7619F76C3]\n\t(No symbol) [0x00007FF7619C0490]\n\t(No symbol) [0x00007FF7619C1743]\n\tGetHandleVerifier [0x00007FF761EE436D+3525677]\n\tGetHandleVerifier [0x00007FF761EF7F3B+3606523]\n\tGetHandleVerifier [0x00007FF761EECEE3+3561379]\n\tGetHandleVerifier [0x00007FF761C57C0A+853194]\n\t(No symbol) [0x00007FF761AF990F]\n\t(No symbol) [0x00007FF761AF5674]\n\t(No symbol) [0x00007FF761AF5816]\n\t(No symbol) [0x00007FF761AE4D89]\n\tBaseThreadInitThunk [0x00007FF8EA71E8D7+23]\n\tRtlUserThreadStart [0x00007FF8EC15BF6C+44]\n\n", "trace": "  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"E:\\AutoTest\\FgAI\\test_001\\test_DossierU.py\", line 29, in setup\n    self._click_upload_button()\n  File \"E:\\AutoTest\\FgAI\\test_001\\test_DossierU.py\", line 44, in _click_upload_button\n    upload_button = self.wait.until(EC.element_to_be_clickable((\n  File \"D:\\Python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "attachments": [{"name": "setup_failed", "source": "c0e02420-3748-4026-9c9c-87a022d24123-attachment.png", "type": "image/png"}], "start": 1742903675469, "stop": 1742903685919}], "start": 1742903675469, "stop": 1742903685989}